src/components/ListItem.jsx

```diff
@@ -1,5 +1,74 @@
 import './ListItem.css';
+import { updateItem } from '../api/firebase.js';
+import { useState, useEffect, useMemo } from 'react';
+import { ONE_DAY_IN_MILLISECONDS } from '../utils/dates.js';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({ listPath, item }) {
+	/* Returns a boolean that is passed into isChecked useState
+	On render, box is checked if purchased less than a day ago */
+
+	const purchasedOneDayAgo = useMemo(() => {
+		if (item.dateLastPurchased !== null) {
+			const timeDiff = Date.now() - item.dateLastPurchased.seconds * 1000;
+			if (timeDiff <= ONE_DAY_IN_MILLISECONDS) {
+				return true;
+			} else {
+				return false;
+			}
+		} else {
+			return false;
+		}
```

Comment:
```suggestion
		if (item.dateLastPurchased === null) {
			return false;
		}

		const timeDiff = Date.now() - item.dateLastPurchased.seconds * 1000;
		return timeDiff <= ONE_DAY_IN_MILLISECONDS;
```

Reply:
Oh, I like that idea! Definitely removes some unnecessary lines of code! I think originally we had used a ternary operator, but it was making it confusing to think about what was being returned. I think this method is both clearer and concise, I'll add this to our PR!